diff -Naur xspeakfree-0.8.1/bin/xspeakfree xspeakfree-0.8.1-p/bin/xspeakfree
--- xspeakfree-0.8.1/bin/xspeakfree	Mon Nov  1 03:26:45 1999
+++ xspeakfree-0.8.1-p/bin/xspeakfree	Thu Nov 16 13:34:26 2000
@@ -1,17 +1,3 @@
 #!/bin/sh
-
-LIB=`dirname $0`/../lib/xspeakfree
-
-wish "$LIB/xspeakfree.tcl" -- "$@" &
-trap "/bin/sh -c \"kill $!
-	kill -9 \`cat /tmp/tclsh-exec.$!.*.pid\`
-	rm /tmp/tclsh-exec.$!.*
-	exit
-	\" >/dev/null 2>&1
-	" 1 2 3 6 14 15 16 17
-pid=$!
-wait $pid
-/bin/sh -c "
-	kill -9 \`cat /tmp/tclsh-exec.$!.*.pid\`
-	rm /tmp/tclsh-exec.$!.*
-" >/dev/null 2>&1
+LIB="`dirname $0`/../lib/xspeakfree"
+wish "$LIB/xspeakfree.tcl" -- "$@" 
